ui/src/helpers/socketConnection.ts

```diff
@@ -0,0 +1,12 @@
+import io from "socket.io-client";
+
+import { MODE } from "constants/env";
+
+export const socket = io(
+  process.env.NODE_ENV === MODE.DEVELOPMENT
+    ? "http://localhost:8000"
+    : "https://api.stellar-anchor-tests.com",
```

Comment:
I just put in this second URL to illustrate how this should be used. Will need to be updated